client/orm/orm_queryset.go

```diff
@@ -64,22 +64,21 @@ func ColValue(opt operator, value interface{}) interface{} {
 
 // real query struct
 type querySet struct {
```

Comment:
We should not using `context.Context` as field, it's not good practice.

And for this structure, the key point is to change `QuerySeter` interface. There are many methods in this interface but most of them don't need the `context` parameter.

But the "teminal method", which means that actually do query to DB, need to add parameter `ctx context.Context` to its signature.

Those methods are changed in this file now, you could check it.

But in order to keep compatiable, we should think about add new method, for example:
```go
Exist()
ExistWitCtx()
```

Reply:
In some aspects, I agree with you. But idk what's the plan for your team on this orm module. Your comment seems inconsistent with the existing code. 

You can see there's already `QueryM2MWithCtx` and `QueryTableWithCtx` method in the codebase.  If we remove `ctx` field on `querySet` struct. These two methods and related code also need to be removed. I reserved the `ctx` just for compatibility with the existing code.

I can do a revise as you suggested. But just wanna make sure what the idea of the guy who wrote and merged `QueryM2MWithCtx` and `QueryTableWithCtx`.

Reply:
let me check that.

Reply:
I believe this is my mistake. 
I think one way is that keep both two methods and your solution is acceptable. But as I said that, using `ctx` as structure's field is not a good practice.
Another way is that mark both `QueryM2MWithCtx` and `QueryTableWithCtx` as deprecated. In this case, we want you remove the ctx field.

What do you think?

Reply:
If we think querySeter is "request-scope", using ctx as structure field looks not bad.

Reply:
To me, I'd prefer the suggestion on the first comment you make. That is to make `context.Context` parameter on the `Insert/Delete/Update` method on `QuerySetter`. I think it's good for any use case cause we don't know how users gonna use it in "request-scope" or something other. 

For the compatibility side. I saw these two `QueryM2MWithCtx ` and `QueryTableWithCtx ` methods are already in the master branch and some tagged versions. I'll push another commit to remove all the `ctx` filed and mark that two methods as deprecated.